pulseapi/entries/urls.py

```diff
@@ -31,6 +32,11 @@
         name='moderate'
     ),
     url(
+        r'^(?P<entryid>[0-9]+)/featured/?',
+        toggle_featured,
+        name='featured'
```

Comment:
this `name` is used to resolve urls, so as this is a toggle, keeping it a verb is a good idea. Let's change this to `feature` instead of `featured`